lib/reek/smells/uncommunicative_module_name.rb

```diff
@@ -21,13 +21,13 @@ class UncommunicativeModuleName < SmellDetector
       # The name of the config field that lists the regexps of
       # smelly names to be reported.
       REJECT_KEY = 'reject'
-      DEFAULT_REJECT_SET = [/^.$/, /[0-9]$/]
+      DEFAULT_REJECT_PATTERNS = [/^.$/, /[0-9]$/]
 
       # The name of the config field that lists the specific names that are
       # to be treated as exceptions; these names will not be reported as
       # uncommunicative.
       ACCEPT_KEY = 'accept'
-      DEFAULT_ACCEPT_SET = ['Inline::C']
+      DEFAULT_ACCEPT_PATTERNS = [/Inline::C/]
```

Comment:
Hmm, since it was there already I assumed the initial author had put some thought into it ;)
@mvz maybe you know more about this?